internal/pkg/service/common/task/key.go

```diff
@@ -7,10 +7,20 @@ import (
 )
 
 type Key struct {
-	ProjectID keboola.ProjectID `json:"projectId" validate:"required"`
-	TaskID    ID                `json:"taskId" validate:"required"`
+	// SystemTask means an internal task that is not scoped to a project, for example "tasks.cleanup"
+	SystemTask bool              `json:"systemTask,omitempty"`
+	ProjectID  keboola.ProjectID `json:"projectId,omitempty" validate:"required_without:SystemTask"`
+	TaskID     ID                `json:"taskId" validate:"required"`
```

Comment:
Zistil som, ze v telemetrii nevidim `tasks.cleanup` task.
- Je to preto lebo je trocha specialny.
- Span sa tam vytvaral, ale metriky chybali.

V com je specialny?
- Je to interny task, ktory nie je viazany na konkretny projekt, vsetky ostatne maju `projectID`.
- Preto sa spustal inak, a prakticky tam bola cast kodu duplicitna.

Preco?
- Riesil to vtedy este Jakub a nenapadlo nas ako to spravit lepsie.
- No ako som teraz isiel fixnut tie metriky, tak mi napadlo elegantne riesenie.

Riesenie:
- Oznacim task ako `SystemTask` a nevyzadujem `ProjectID`, vid validaciu `required_without`.
- Duplicitny kod tak pojde prec + automaticky tym vyriesim problem s metrikami.